packages/playwright/src/mcp/test/testContext.ts

```diff
@@ -227,7 +228,15 @@ export class TestContext {
     };
 
     try {
-      const reporter = new ListReporter({ configDir, screen, includeTestId: true });
+      const reporter = new Multiplexer([
+        new ListReporter({ configDir, screen, includeTestId: true }),
+        {
+          version: () => 'v2',
+          onTestPaused() {
+            return new Promise(() => {}); // stall
```

Comment:
I don't quite get this part. Why do we need to stall here? Let's make reporter's `onTestPaused()` not called when running in a test runner mode for now.